File: profiles/cclcc/hud/tipsnotification.lua
root.TipsNotification = {
    Type = TipsNotificationType.CCLCC,
    TextTableId = 0,
    NotificationTextPart1MessageId = 100,
    NotificationTextPart2MessageId = 101,
    FadeInDuration = 0.5,
    FadeOutDuration = 0.5,
    NotificationBackground = "TipNotificationBackground",
    BackgroundPositionX = 571,
    BackgroundPositionYOffset = 55,
    NotificationPositionX = 600,
    NotificationPositionYOffset = 10,
    TimerDuration = 2,
    MoveAnimationDuration = 1,
    FontSize = 26,
    TipNameColor = { 0x06a4ff, 0x035280 },
    NotificationTextColor = { 0x080808, 0x000000 },
};

root.Sprites["TipNotificationBackground"] = {
    Sheet = "Data",
    Bounds = { X = 1459, Y = 1, Width = 817, Height = 120 },
};

File: src/profile/games/cclcc/tipsnotification.cpp
#include "tipsnotification.h"
#include "../../../log.h"
#include "../../profile_internal.h"

namespace Impacto {
namespace Profile {
namespace CCLCC {
namespace TipsNotification {

void Configure() {
  NotificationBackground = EnsureGetMember<Sprite>("NotificationBackground");

  BackgroundPositionX = EnsureGetMember<float>("BackgroundPositionX");
  BackgroundPositionYOffset =
      EnsureGetMember<float>("BackgroundPositionYOffset");
  NotificationPositionX = EnsureGetMember<float>("NotificationPositionX");
  NotificationPositionYOffset =
      EnsureGetMember<float>("NotificationPositionYOffset");
  TimerDuration = EnsureGetMember<float>("TimerDuration");

  FontSize = EnsureGetMember<float>("FontSize");
  TipNameColor = EnsureGetMember<DialogueColorPair>("TipNameColor");
  NotificationTextColor =
      EnsureGetMember<DialogueColorPair>("NotificationTextColor");
}

}  // namespace TipsNotification
}  // namespace CCLCC
}  // namespace Profile
}  // namespace Impacto